### PR TITLE
fix: docker compose dev for arm machine

### PR DESCRIPTION
**Pull request by abuaboud**
## What is the purpose of this PR?

This PR is intended to resolve the issues discussed in the pull request and its associated comments (https://github.com/activepieces/activepieces/pull/797)

1- The ownership of the node_modules files is no longer assigned to the root user.
2- The npm ci command is now being executed inside the development container designed for arm-based systems.
3- The isolate arm feature is no longer utilized due to its requirement for root privileges.

Testing:
To ensure the efficacy of these changes, I deleted the node_modules directory and ran the activepieces program. Subsequently, I tested a flow webhook and code piece.

### CODE REVIEW

**Comment by nileshtrivedi** [comment]
I was able to boot the app from the `fix/arm-docker-compose` branch with `docker compose -p apdev -f docker-compose.dev.yml up --build`. I created a user account in SANDBOXED mode (by modifying AP_EXECUTION_MODE and AP_SIGN_UP_ENABLED in `docker-compose.dev.yml`) and then switched back to UNSANDBOXED to be able to run Code blocks.

Thanks for this change. This enables me to develop on Apple M1 machines. 👍

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -55,7 +35,6 @@ services:
       - postgres_data_dev:/var/lib/postgresql/data
     networks:
       - activepieces_dev
-#    command: -c log_statement=all -c shared_preload_libraries=auto_explain -c auto_explain.log_min_duration=0
```
Comment (khaledmashaly) [inline docker-compose.dev.yml:39]: Can you please revert this line? It's helpful in debugging Postgres queries, but it produces a lot of logs that's why it's commented by default :sweat_smile:

**Review by khaledmashaly — DISMISSED** [review]
EXCELLENTE!
This works very well, thank you!

**Comment by abuaboud** [comment]
> I was able to boot the app from the `fix/arm-docker-compose` branch with `docker compose -p apdev -f docker-compose.dev.yml up --build`. I created a user account in SANDBOXED mode (by modifying AP_EXECUTION_MODE and AP_SIGN_UP_ENABLED in `docker-compose.dev.yml`) and then switched back to UNSANDBOXED to be able to run Code blocks.
> 
> Thanks for this change. This enables me to develop on Apple M1 machines. +1

Updated the docs with default test credentials, good catch!

**Pull request merged**